README.md

```diff
@@ -0,0 +1,52 @@
+# aws-lambda-batch-invoker
+
+Module that allows batch invoking a lambda function synchronously via the `RequestResponse` invocationType or asynchronously via the `Event` invocationType.
+
+# Usage
+
+## new BatchInvoker(options)
+Constructor for the class.
+- **options** - { Object, optional } - an object to provide configuration options.
+  - **logger** - { Object/Function, optional } - a valid logger. If no logger is provided, no debug information or errors will be logged.
+
+```javascript
+const Invoker = require('aws-lambda-batch-invoker');
+const logger = require('./path/to/your/logger');
+
+// no logger
+const invoker = new Invoker();
+
+// logger as option
+const invoker2 = new Invoker({ logger });
+
+// override class defaults with options
+const invoker2 = new Invoker({ isAsync: true, maxConcurrency: 10 });
+
+```
+
+## invoke(functionName, payloads, clientContext, options, cb)
```

Comment:
if we were to do that, I'd like to do it to the lambda-invoke library as well. I think it could be nice as it matches how aws actually does its own invoke method, and makes it easier to add new params without introducing breaking changes